### PR TITLE
modal style, still need to format date

### PR DESCRIPTION
**Pull request by Nicola-Carroll**
just a bit of styling

### CODE REVIEW

File: client/src/index.css [hunk 1]
```diff
@@ -31,3 +31,8 @@ code {
   height: 60vh;
   max-height: 100%;
 }
+
+#claim-button {
+  background-color: #7ed957;
+  margin-left: 3px;
+}
```
Comment (yousufmohamed17) [inline client/src/index.css:38]: So this overwrites the bootstrap styling on the button and personally I think it looks better without it so I'd get rid of this. Currently it's a different shade of green all the other buttons.

Comment (Nicola-Carroll) [reply client/src/index.css:38]: before it was a kinda ugly grey i believe

File: client/src/components/shared/ModalContent.component.js [hunk 2]
```diff
@@ -30,25 +31,29 @@ export default class ModalContent extends Component {
...
-        <p>
-          There are {this.props.content.listingNumberOfMeals} meals available
-        </p>
-        <p>Details below: </p>
-        <p>{this.props.content.listingDescription}</p>
-        <p>
-          Grab this donation before {this.props.content.listingAvailableTill}
+        <p class="font-weight-bold">
+          There are {this.props.content.listingNumberOfMeals} meals available to
+          collect until {this.props.content.listingAvailableTill}.
```
Comment (yousufmohamed17) [inline client/src/components/shared/ModalContent.component.js:36]: Replace with this to get Lizzy's formatting `{Date(this.props.content.listingAvailableTill)}`

File: client/src/index.css (hunk 1 above)
Comment (yousufmohamed17) [reply client/src/index.css:38]: Oh but this styling is applying to the confirm button not the go back button.

Comment (Nicola-Carroll) [reply client/src/index.css:38]: good point i htought i had it the other way

**Pull request merged**